### ISSUE TITLE
Pagination: improve accessibility for disabled links

### ISSUE THREAD

**Issue by thbt** [twbs/bootstrap#22009]
Using a screen reader, disabled pagination links are not advertised as "disabled". They can't be focused but the screen reader will still read them.

I would suggest adding either `<span class="sr-only">disabled</span>` or `aria-disabled="true"` to the link.

Let me know what you think.

**Comment by thbt** [comment]
One more thing : even without any vision impairment, it's hard to notive a difference between a disabled link and a enabled one.

**Comment by thbt** [comment]
I'm just bumping up this issue since I updated my recommendations

**Comment by thbt** [comment]
One more thing, you suggest swapping out `<a>` for `<span>`, but I would like to know which solution is the best for users of assistive technology.

**Comment by thbt** [comment]
After testing and discussing it I chose to go with the `span` instead of a disabled anchor. In my opinion we lose the semantic meaning of a disabled link, but the user doesn't need to think why the link is disabled, and some screen readers doesn't announce the anchor as disabled (i.e. ChromeVox)
